deploy.sh

```diff
@@ -2,23 +2,26 @@
 
 # Install dependencies
 yarn;
-yarn global add mup https://github.com/jimrandomh/mup-aws-beanstalk;
+yarn global add mup@1.4.6 https://github.com/jimrandomh/mup-aws-beanstalk;
 yarn global add json;
 yarn global add @sentry/cli;
 
 # Variables & Arguments
 settings_path="$1";
 deploy_config_path="$2";
-sentry_version=$(sentry-cli releases propose-version);
+echo 'deploy_config_path'
+# sentry_version=$(sentry-cli releases propose-version);
 
+git clone https://github.com/elasticdog/transcrypt.git
 # Environment variables and secrets provided via travis-ci
-git clone https://Discordius:$GITHUB_TOKEN@github.com/LessWrong2/LessWrong-Credentials.git
+./transcrypt/transcrypt -c aes-256-cbc -p $TRANSCRYPT_SECRET -y
 
-# Set up sentry configurations
-sentry-cli releases new -p lesswrong $sentry_version;
-sentry-cli releases set-commits --auto $sentry_version;
-json -I -f $settings_path -e "this.public.sentry.release=\"$sentry_version\"";
+# # Set up sentry configurations
+# sentry-cli releases new -p lesswrong $sentry_version;
+# sentry-cli releases set-commits --auto $sentry_version;
+# json -I -f $settings_path -e "this.public.sentry.release=\"$sentry_version\"";
 
 # Deploy
 cp $deploy_config_path ./config.secret; # Have to copy this because mup deploy resolves in directory relative to config file
-bash ./travis_wait "mup deploy --config config.secret --settings $settings_path";
\ No newline at end of file
+
+bash ./travis_wait "mup deploy --config config.secret --settings $settings_path";
```

Comment:
Should `config.secret` be `$deploy_config_path` here instead of hardcoded?

Reply:
The previous line makes this line not-broken. For us we don't actually care about "copying this because mup deploy resolves in directory relative to config file". Because unlike LW our config file starts out in this directory. But we do care about minimally differing with lesswrong code. Given that we haven't changed this or the previous line, I definitely want to leave it as-is.